systests/microprofile/pom.xml

```diff
@@ -42,6 +42,7 @@
         <cxf.arquillian.version>1.1.14.Final</cxf.arquillian.version>
         <cxf.slf4j.version>1.7.25</cxf.slf4j.version>
         <cxf.commons.logging.version>1.2</cxf.commons.logging.version>
+        <cxf.jetty.version>9.4.14.v20181114</cxf.jetty.version>
```

Comment:
Can we import some of these properties from the CXF parent pom as well? For example, Jetty, Johnzon, Weld, SLF4J, etc.

Reply:
Hi @coheigea - I'll try to consolidate the Johnson, Weld, SLF4J, etc. into the parent pom.  There is a particular problem with newer versions of Jetty - to workaround it, I think we might need to keep the (downlevel) jetty version specified in the MP systest. The problem is described in issue https://github.com/eclipse/jetty.project/issues/3656 - since it only seems to affect the MP Rest Client TCKs, I didn't want to downgrade it for everybody.

Reply:
Sounds good @andymc12 . Re the Jetty issue, I recently upgraded CXF and worked around the SSL client auth issue - see here: https://github.com/apache/cxf/commit/af6a95f5b7efc141cc55792e10d05d471905cf32#diff-d0e45878043844ffc41aac437e86b602

Reply:
Thanks @coheigea - We will update the MP Rest Client TCK with that change (I verified that it works locally).  If it's ok with you, I'd still like to leave this older version in (for the MP Rest Client tests only) until we release a new version of the TCK with these changes.